mavros/launch/px4_config.yaml

```diff
@@ -68,7 +68,7 @@ imu:
 local_position:
   frame_id: "map"
   tf:
-    send: false
+    send: true
```

Comment:
I don't support this being enabled by default.

Reply:
This can have unexpected repercussions for systems where the map transform is already published by something other than mavros. Please revert, and we can put in a doc note.